test-data/unit/pythoneval.test

```diff
@@ -644,7 +644,7 @@ def f(*args: str) -> str: return args[0]
 map(f, ['x'])
 map(f, [1])
 [out]
-_program.py:4: error: Argument 1 to "map" has incompatible type "Callable[[VarArg(str)], str]"; expected "Callable[[int], str]"
+_program.py:4: error: Argument 1 to "map" has incompatible type "def f(*args: str) -> str"; expected "Callable[[int], str]"
 
```

Comment:
Two formats in one error message.

Reply:
FWIW I would prefer to always use same format in same error message (if it is easy to implement).